app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+class CommentsController < ApplicationController
+  include CommentsHelper
+  before_action :authenticate_user!
+
+  def create
+    @comment = @commentable.comments.new comment_params
```

Comment:
ALWAYS use "()" for methods and params.
ALWAYS. it's really helping with code readability